docs/resourcesCategory/resourcesCategory.yaml

```diff
@@ -50,7 +50,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/Error'
+                $ref: "#/components/Error"
```

Comment:
same here, it should be '', check linter with prettier

Reply:
> same here, it should be '', check linter with prettier

yup, checked it, and you are right. Thanks! Fixed it